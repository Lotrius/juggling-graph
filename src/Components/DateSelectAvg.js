import React, { Component } from 'react';
import DatePicker from 'react-datepicker';
import "react-datepicker/dist/react-datepicker.css";

var d = new Date();
d.setMonth(d.getMonth() - 1);

class DateSelectAvg extends Component {
    render() {
        const { averageDate } = this.props;
        return (
            <div >
                <DatePicker
                    selected={averageDate}
                    onChange={date => this.props.getAverageData(date)}
                    dateFormat="MM/yyyy"
                    showMonthYearPicker
                />
            </div>

        );
    }
}

export default DateSelectAvg;