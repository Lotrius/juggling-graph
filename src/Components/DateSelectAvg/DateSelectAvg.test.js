import { shallow } from 'enzyme';
import React from 'react';
import DateSelectAvg from './DateSelectAvg';

it('Render DateSelectAvg component', () => {
  expect(shallow(<DateSelectAvg />)).toMatchSnapshot();
});
