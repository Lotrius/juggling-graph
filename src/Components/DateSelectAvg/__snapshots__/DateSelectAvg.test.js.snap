// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render DateSelectAvg component 1`] = `ShallowWrapper {}`;
